.travis.yml

```diff
@@ -33,6 +33,8 @@ install:
       pip install -r requirements2.txt;
     else
       pip install -r requirements3.txt;
+      wget http://launchpadlibrarian.net/151932048/libc6_2.17-0ubuntu5.1_i386.deb;
```

Comment:
Thanks! This will be a problem for future Perry to deal with haha. But travis has been pretty annoying as well as Python2/Qt4 compatibility.